packages/wifi_iot/android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -391,10 +403,20 @@ private void isSSIDHidden(Result poResult) {
 
       poResult.error("Exception [isSSIDHidden]", "Wifi AP not Supported", null);
     } else {
-      poResult.error(
-          "Exception [isSSIDHidden]",
-          "Getting SSID visibility is not supported on API level >= 26",
-          null);
+      if (apReservation != null) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+          SoftApConfiguration softApConfiguration = apReservation.getSoftApConfiguration();
+          poResult.success(softApConfiguration.isHiddenSsid());
+        } else {
+          poResult.success(true);
```

Comment:
```suggestion
          poResult.success(false);
```

My bad, I meant false!

Reply:
No problem 😉